res/database/Default/database_people.xml

```diff
@@ -257,1017 +276,1295 @@
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Lee</first_name_original>
+			<last_name_original>van Cleeff</last_name_original>
+			<details job="2" gender="0" birthday="1925-01-09" deathday="1989-12-16" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="ab39af52-afe6-4e11-8e4f-5b8c2ceb59e2" tmdb_id="0" imdb_id="nm0001884" creator="">
 			<first_name>Max</first_name>
 			<last_name>von Sydowson</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Max</first_name_original>
+			<last_name_original>von Sydow</last_name_original>
+			<details job="2" gender="0" birthday="1929-04-10" deathday="2020-03-08" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="f0538156-247f-45f6-a710-39c66bd7a9f9" tmdb_id="0" imdb_id="nm0938754" creator="">
 			<first_name>Rolf</first_name>
 			<last_name>Walther</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Ralf</first_name_original>
+			<last_name_original>Wolter</last_name_original>
+			<details job="2" gender="0" birthday="1926-11-26" deathday="2022-10-14" country="" />
 		</person>
 		<person id="88715556-4743-44c1-9e1e-99999e4c753f" tmdb_id="0" imdb_id="nm0001835" creator="">
 			<first_name>Carl</first_name>
 			<last_name>Weathehs</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Carl</first_name_original>
+			<last_name_original>Weathers</last_name_original>
+			<details job="2" gender="0" birthday="1948-01-14" deathday="" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="ef2632a4-e8e9-4faf-b8dc-55ba43822c9d" tmdb_id="0" imdb_id="nm0000244" creator="">
 			<first_name>Signme</first_name>
 			<last_name>Weber</last_name>
 			<nick_name></nick_name>
-			<details job="2" gender="0" birthday="" deathday="" country="" />
+			<first_name_original>Sigourney</first_name_original>
+			<last_name_original>Weaver</last_name_original>
+			<details job="2" gender="0" birthday="1949-10-08" deathday="" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
-		<person id="817052f9-cfbd-415c-a4b4-a225011afcb1" tmdb_id="0" imdb_id="" creator="">
+		<person id="817052f9-cfbd-415c-a4b4-a225011afcb1" tmdb_id="0" imdb_id="" creator="" comment="parody version of cd3a2e20-efe1-4a4c-9249-164b67d185f7">
```

Comment:
@GWRon : should we prevent duplicates of non-fictional persons

Reply:
This "Weissenacker" is used in fictional programmes, the "real" one in non-fictional programmes, so I guess the differentiation already was on-purpose and desired. Generally I think it's fine to have separate parody versions of "real" people to use for some special jokes or occasions.